docs/conf/helpop-full.conf.example

```diff
@@ -752,7 +753,7 @@ Sends a message to all +w users.">
 
 <helpop key="rline" value="/RLINE <regex> [<duration> :<reason>]
 
-Sets or removes an r-line (regex line) on a n!u@h\\sgecos mask. You
+Sets or removes an r-line (regex line) on a n!u@h\sgecos mask. You
```

Comment:
I have tested that this shows correctly when called via `/helpop rline` (in Quassel). So unless other clients parse it incorrectly, I assume it was a typo and can confuse people as to whether a `\s` or a `\\s` is required in the mask.

Reply:
This is an escape sequence when using `<config format="compat">`. We should probably force XML mode in these example files.

Reply:
Ah compat format! Well the examples do ship as `xml` with `prefixquit="""` and a note about it. Could add a comment note before the rline key.